meson.build

```diff
@@ -1494,15 +1508,17 @@ if opt and not is_disabler(exe)
   bashcompletions += ['tunelp']
 endif
 
+opt = not get_option('build-fstrim').require(build_libmount).disabled()
 exe = executable(
   'fstrim',
   fstrim_sources,
   include_directories : includes,
-  link_with : [lib_common,
-               lib_mount],
+  link_with : [lib_common],
+  dependencies : [mount_dep],
```

Comment:
Moving everything to dependencies alone looks like a good preparatory commit.
Then it's more clear what is actually needed.

Reply:
I've done this now for both `libblkid` and `libmount`, each in a separate commit. This keeps them consistent.